cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceRuntimeService.java

```diff
@@ -773,8 +774,19 @@ public boolean accept(String className, URL classUrl, URL classPathUrl) {
       LOG.warn("Not including HBaseTableUtil classes in submitted Job Jar since they are not available");
     }
 
+    // Add the logback.xml as a resource while creating the MapReduce Job JAR
+    Set<URI> logbackURI = Sets.newHashSet();
+    try {
+      URL logback = getClass().getResource("logback.xml");
```

Comment:
This will be made available by the AbstractDistributedProgramRunner (getLogbackURI method) launches the MapReduceTwillApp ?